src/odemis/acq/calibration.py

```diff
@@ -316,7 +315,7 @@ def apply_spectrum_corrections(data, bckg=None, coef=None):
             try:
                 wl_bckg = spectrum.get_wavelength_per_pixel(bckg)
             except KeyError:
-                raise ValueError("Found no MD_WL_* metadata in background image.")
+                raise ValueError("Found no MD_WL_LIST metadata in background image.")
```

Comment:
Maybe adding a reason why that is bad to the error message would be nice.
I don't know the code/context but something like:
> wavelenght information is required to do XXXXX"?

Reply:
That's quite a little bit out of scope from this task.
The error message is not really intended to the end-user. The GUI could catch it, and report the background data is not fitting. That said, I don't think it's handled for now. You could make the message indeed a little less cryptic by writing "Found no spectrum metadata (MD_WL_LIST) in the background image".